### PR TITLE
bubble chart

### PR DESCRIPTION
**Pull request by oriv**
## Status
READY

https://www.figma.com/file/lEQWuPmFC650mWWL9Z0yQe/Manage-Performance?node-id=365%3A26

### CODE REVIEW

**Review by harellevy — APPROVED** [review]

File: projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble.chart.component.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  NgZone,
+} from '@angular/core';
+import { ChartCore } from '../chart/chart-core';
+import { SeriesBubbleOptions } from 'highcharts';
+import { ChartTypesEnum } from '../charts.enum';
+
+@Component({
+  selector: 'b-bubble-chart',
+  templateUrl: '../chart/chart.component.html',
+  styleUrls: ['../chart/chart.component.scss'],
```
Comment (harellevy) [inline projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble.chart.component.ts:16]: missing onpush

File: projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble-chart.stories.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+          '#BF8A78',
+          '#C0755A',
+          '#866161',
+          '#663E4E',
+          '#574285',
+          '#6969C6',
+          '#556E8A',
+          '#789BC2',
+          '#9BC7FA',
+          '#6DC3BC',
```
Comment (harellevy) [inline projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble-chart.stories.ts:132]: does this chart uses color pallate?

File: projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble-chart.stories.ts [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  withKnobs
+);
+const template = `<b-bubble-chart
+        [data]="data"
+        [preTooltipValue]="preTooltipValue"
+        [postTooltipValue]="postTooltipValue"
+        [showDataLabels]="showDataLabels"
+        [legend]="legend"
+        [legendPosition]="legendPosition"
+        [colorPalette]="colorPalette"
```
Comment (harellevy) [inline projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble-chart.stories.ts:27]: maybe we can remove if not in use

File: projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble-chart.stories.ts (hunk 2 above)
Comment (oriv) [reply projects/ui-framework/bob-charts/src/charts/bubble-chart/bubble-chart.stories.ts]: no

**Review by galloween — APPROVED** [review]

**Pull request merged**